2 entries for - Q. How are rules processed?

CHDAFNI-MSFT opened the pull request:
2 entries and different answers for: Q. How are rules processed?
Article should remove the erroneous one

azurecla commented:
Hi __@tudordid__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (tudordid). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@bradsev said he was reviewing today.

bradsev commented:
Accept this pull request.

CHDAFNI-MSFT committed:
2 entries for - Q. How are rules processed?

2 entries and different answers for: Q. How are rules processed?
Article should remove the erroneous one
